java/src/main/java/lesson03/solution/Formatter.java

```diff
@@ -32,12 +35,13 @@ private Formatter(Tracer tracer) {
 
         @GET
         public String format(@QueryParam("helloTo") String helloTo, @Context HttpHeaders httpHeaders) {
-            try (Scope scope = Tracing.startServerSpan(tracer, httpHeaders, "format")) {
```

Comment:
So far in the tutorial, there's no mention about the `Tracing.startServerSpan()` method, so, I think it might be best to have duplicated code here.

Reply:
I think the tutorial hasn't been updated to the ScopeManager extension

Reply:
Why not introduce that function first, maybe as static in the Hello class? Seems like a bad practice to show tutorial with repeated code.

Reply:
I added a commit to this PR with this change. How does it look?